core/src/main/java/org/mskcc/cbio/portal/scripts/ResetDatabase.java

```diff
@@ -59,9 +59,10 @@ public static void resetAnySizeDatabase() throws DaoException {
         DaoCancerStudy.deleteAllRecords();
         DaoGeneOptimized daoGene = DaoGeneOptimized.getInstance();
         daoGene.deleteAllRecords();
+        DaoGeneset.deleteAllRecords();
```

Comment:
Yes indeed the `DaoGenesetInfo` should also be reset here. Will fix it!

The `DaoGenesetHierarchy` is actually reset in `DaoGeneset.deleteAllRecords()`, and the `DaoGenesetHierarchyLeaf` records are removed when records from `DaoGenesetHierarchy` are removed. There might be a better way to make this clear. I'll discuss it with @pieterlukasse